services/static-webserver/client/source/class/osparc/dashboard/CardBase.js

```diff
@@ -768,13 +776,22 @@ qx.Class.define("osparc.dashboard.CardBase", {
     },
 
     __applyState: function(state) {
-      let lockInUse = false;
+      let projectInUse = false;
       if ("locked" in state && "value" in state["locked"]) {
-        lockInUse = state["locked"]["value"];
+        projectInUse = state["locked"]["value"];
       }
-      this.setBlocked(lockInUse ? "IN_USE" : false);
-      if (lockInUse) {
-        this.__showBlockedCardFromStatus("IN_USE", state["locked"]);
+
+      if (osparc.utils.DisabledPlugins.isSimultaneousAccessEnabled()) {
+        if (projectInUse && state["locked"]["status"] === "OPENED") {
+          this.__showWhoIsIn(state["locked"]["owner"]);
+        } else {
+          this.__showWhoIsIn(null);
+        }
+      } else {
+        this.setBlocked(projectInUse ? "IN_USE" : false);
+        if (projectInUse) {
+          this.__showBlockedCardFromStatus("IN_USE", state["locked"]);
+        }
```

Comment:
here I will return a list of users instead of just one.
Nevertheless I am not sure you need that specific change, since I was able to do it backend only. will let you know once it is ready.

Reply:
I mentioned [here](https://github.com/ITISFoundation/osparc-issues/issues/1961) that I would need the ``group_id`` instead of the ``user_id``